site/pt/blog/new-in-devtools-93/index.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: "layouts/blog-post.njk"
+title: "O que há de novo no DevTools (Chrome 93)"
+authors:
+  - jecelynyeen
+date: 2021-07-28
+updated: 2021-07-28
+description:
+  "Consultas de contêiner CSS editáveis, prévia de web-bundles, melhor manejo de strings no Console e muito mais."
+hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/TiDoAuzLse9uUSoylxNA.jpg'
+alt: ''
+tags:
+  - new-in-devtools
+  - devtools
+  - chrome-93
+---
+
+*Tradução realizada por [Alvaro Camillo Neto](https://www.linkedin.com/in/alvarocamillont/).*
+
+{% include 'partials/devtools/pt/banner.md' %}
+
+{% YouTube id="1VaPAnUGRz8" %}
+
+## Consultas de contêiner CSS editáveis no painel Styles {: #container-queries }
+
+Agora você pode visualizar e editar [Consultas de contêiner CSS](https://web.dev/new-responsive/#responsive-to-the-container) no painel **Styles**.
+
+As consultas de contêiner fornecem uma abordagem muito mais dinâmica para design responsivo. A regra `@container` funciona de maneira semelhante a uma consulta de mídia com `@media`. No entanto, em vez de consultar a viewport e o agente do usuário em busca de informações, `@container` consulta o container ancestral que corresponde a certos critérios.
+
+No painel **Elements**, clique em um elemento DOM com a regra `@container`, DevTools agora exibe as informações `@container` no painel **Styles**. Clique nele para editar o tamanho. O painel **Styles** também exibe as informações do contêiner correspondente. Passe o mouse sobre ele para destacar o elemento do contêiner na página e verifique o tamanho do contêiner. Clique nele para selecionar o elemento do contêiner.
```

Comment:
nit: Start a new sentece with "O Devtools".
nit: Avoid some repetition of "contêiner": "Passe o mouse sobre ele para destacar o elemento do contêiner na página e verifique o seu tamanho".

```suggestion
No painel **Elements**, clique em um elemento DOM com a regra `@container`. O DevTools agora exibe as informações `@container` no painel **Styles**. Clique nele para editar o tamanho. O painel **Styles** também exibe as informações do contêiner correspondente. Passe o mouse sobre ele para destacar o elemento do contêiner na página e verifique o tamanho do contêiner. Clique nele para selecionar o elemento do contêiner.
```